.github/labels.yml

```diff
@@ -1,100 +1,23 @@
-  color: "5319E7"
-- name: "component / contrib.auth"
-  color: "5319E7"
-- name: "component / contrib.contenttypes"
-  color: "5319E7"
-- name: "component / contrib.flatpages"
-  color: "5319E7"
-- name: "component / contrib.humanize"
-  color: "5319E7"
-- name: "component / contrib.messages"
-  color: "5319E7"
-- name: "component / contrib.postgres"
-  color: "5319E7"
-- name: "component / contrib.redirects"
-  color: "5319E7"
-- name: "component / contrib.sessions"
-  color: "5319E7"
-- name: "component / contrib.sitemaps"
-  color: "5319E7"
-- name: "component / contrib.sites"
-  color: "5319E7"
-- name: "component / contrib.staticfiles"
-  color: "5319E7"
-- name: "component / contrib.syndication"
-  color: "5319E7"
-- name: "component / Core (Cache system)"
-  color: "5319E7"
-- name: "component / Core (Mail)"
-  color: "5319E7"
-- name: "component / Core (Management commands)"
-  color: "5319E7"
-- name: "component / Core (Other)"
-  color: "5319E7"
-- name: "component / Core (Serialization)"
-  color: "5319E7"
-- name: "component / Core (System checks)"
-  color: "5319E7"
-- name: "component / Core (URLs)"
-  color: "5319E7"
-- name: "component / CSRF"
-  color: "5319E7"
-- name: "component / Database layer (models, ORM)"
-  color: "5319E7"
-- name: "component / Documentation"
-  color: "5319E7"
-- name: "component / Error reporting"
-  color: "5319E7"
-- name: "component / File uploads/storage"
-  color: "5319E7"
-- name: "component / Forms"
-  color: "5319E7"
-- name: "component / Generic views"
-  color: "5319E7"
-- name: "component / GIS"
-  color: "5319E7"
-- name: "component / HTTP handling"
-  color: "5319E7"
-- name: "component / Internationalization"
-  color: "5319E7"
-- name: "component / Migrations"
-  color: "5319E7"
-- name: "component / Packaging"
-  color: "5319E7"
-- name: "component / Template system"
-  color: "5319E7"
-- name: "component / Testing framework"
-  color: "5319E7"
-- name: "component / Utilities"
-  color: "5319E7"
-- name: "size / small"
-  color: "C2E0C6"
-- name: "size / medium"
-  color: "C2E0C6"
-- name: "size / large"
-  color: "C2E0C6"
-- name: "phase / new"
-  color: "FBCA04"
-- name: "phase / is there community support"
-  color: "FBCA04"
-- name: "phase / is this expected in core"
-  color: "FBCA04"
-- name: "phase / third-party package"
-  color: "FBCA04"
-- name: "phase / needs community DEP"
-  color: "FBCA04"
-- name: "phase / wontfix"
-  color: "FBCA04"
-- name: "phase / needs SC DEP"
-  color: "FBCA04"
-- name: "phase / implementation"
-  color: "FBCA04"
-- name: "type / commercial"
-  color: "9F2EEB"
-- name: "type / feature"
-  color: "9F2EEB"
-- name: "type / merge-package"
-  color: "9F2EEB"
\ No newline at end of file
+- name: "Django Core"
+  color: "3FBE38"
+  description: "This idea is suitable for inclusion in Django itself"
+- name: "Third Party Package"
+  color: "2898D8"
+  description: "This idea is suitable for the third party package ecosystem"
+- name: "Admin"
```

Comment:
I'm not sure if a random color is going to be used. Maybe it's better to pick one for these to avoid the chance of randomness?

Reply:
I assumed a random color would be used and thought that was probably fine. I'm also OK with using a non-random color, but I'm not sure which one(s)

Reply:
Ah, gotcha. I'll mark this as resolved. I don't have a strong preference.